ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -17129,9 +17155,12 @@ ABoundActivity * HqlCppTranslator::doBuildActivitySOAP(BuildCtx & ctx, IHqlExpre
     if (action)
         doBuildVarStringFunction(instance->startctx, "getSoapAction", action->queryChild(0));
 
+    doBuildHttpHeaderStringFunction(instance->startctx, expr);
+
     IHqlExpression * httpHeader = expr->queryAttribute(httpHeaderAtom);
     if (httpHeader)
```

Comment:
I'll remove.  If someone does copy a new workunit to an old environment they get an error?  That's actually better since the behavior would change subtly if they were sending more than one header and could lead to confusion.